client/app/src/main/res/navigation/registration_navigation.xml

```diff
@@ -14,7 +14,8 @@
             android:id="@+id/action_phoneNumberFragment_to_OTPFragment"
             app:destination="@id/OTPFragment"
             app:enterAnim="@anim/nav_default_pop_enter_anim"
-            app:exitAnim="@anim/nav_default_pop_exit_anim" />
+            app:exitAnim="@anim/nav_default_pop_exit_anim"
+            app:launchSingleTop="true" />
```

Comment:
`launchSingleTop` is kinda useless for us imo. We won't be launching the same fragment again.